docker/spark/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM apache/spark:3.5.1-java17
+
+USER root
+
+# Install Python and required utilities
+RUN apt-get update && \
+    apt-get install -y python3 python3-pip && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install Python packages for Iceberg and AWS
+RUN pip install --no-cache-dir \
+    pyiceberg==0.7.0 \
+    boto3 \
+    requests \
+    pyspark
```

Comment:
we don't seem to be using pyspark, unless some existing code relies on it?

Reply:
Removed it